include/grpc/impl/codegen/grpc_types.h

```diff
@@ -232,8 +232,8 @@ typedef struct {
 /** Secondary user agent: goes at the end of the user-agent metadata
     sent on each request. A string. */
 #define GRPC_ARG_SECONDARY_USER_AGENT_STRING "grpc.secondary_user_agent"
-/** The minimum time between subsequent connection attempts, in ms */
-#define GRPC_ARG_MIN_RECONNECT_BACKOFF_MS "grpc.min_reconnect_backoff_ms"
+/** The minimum time we are giving a connection to complete, in ms */
+#define GRPC_ARG_MIN_CONNECT_TIMEOUT "grpc.min_connect_timeout_ms"
```

Comment:
We probably need to continue to support the old name for at least one release to ensure backward compatibility.  Please re-add the old macro, change the code to look at both of them, and add this to doc/core/pending_api_cleanups.md.